lib/plugins/inspec-reporter-cli/lib/inspec-reporter-cli.rb

```diff
@@ -0,0 +1,13 @@
+require_relative "inspec-reporter-cli/version"
+
+module InspecPlugins
+  module CliReporter
+    class Plugin < ::Inspec.plugin(2)
+      plugin_name :'inspec-reporter-cli'
+      reporter :cli do
+        require_relative "inspec-reporter-cli/reporter"
+        InspecPlugins::CliReporter::Reporter
```

Comment:
One thing I reallly like about getting this conversion done is it'll help give further examples for developers building their own InSpec reporters in the future (including me! 😄 )